src/main/java/fr/theorozier/webstreamer/display/render/DisplayLayerManager.java

```diff
@@ -11,7 +11,7 @@
  * This class is responsible for caching and keeping the number of layer to the minimum.
  */
 @Environment(EnvType.CLIENT)
-public class DisplayLayerManager extends DisplayLayerMap<URI> {
+public class DisplayLayerManager extends DisplayLayerMap<Object> {
```

Comment:
No need to change the type, the main key for the manager should remain the URI.

Reply:
Resolved in https://github.com/mindstorm38/fabric-webstreamer/pull/28/commits/1c0c2e9fdf6e0900dbf82b5f743fdc32ea4a4e03